File: app/src/main/java/com/vcinsidedigital/demo/MainActivity.java
package com.vcinsidedigital.demo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import com.vcinsidedigital.digitalsignature.DigitalSignature;

import java.io.File;

public class MainActivity extends AppCompatActivity
{

    private ImageView imageView;
    private Button btnToSign;
    private String tmpFile;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        imageView = findViewById(R.id.imageView);
        btnToSign = findViewById(R.id.btn_toSign);

        tmpFile = (String) getIntent().getStringExtra("filePath");

        if(tmpFile != null)
        {
            File file = new File(tmpFile);

            Bitmap bitmap = BitmapFactory.decodeFile(file.getAbsolutePath());
            imageView.setImageBitmap(bitmap);
        }

        btnToSign.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                finish();
                Intent intent= new Intent(getApplicationContext(), ToSignActivity.class);
                startActivity(intent);
            }
        });

    }
}

File: demokotlin/src/main/java/com/vcinsidedigital/demokotlin/MainActivity.kt
package com.vcinsidedigital.demokotlin

import android.content.Intent
import android.graphics.Bitmap
import android.graphics.BitmapFactory
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import android.widget.Toast
import java.io.File

class MainActivity : AppCompatActivity()
{
    var imageView:ImageView? = null
    var btnToSign:Button? = null
    var tmpFile:String? = null


    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        imageView = findViewById(R.id.imageView)
        btnToSign = findViewById(R.id.btn_toSign)

        btnToSign?.setOnClickListener {
            finish()
            val intent:Intent = Intent(applicationContext, ToSignActivity::class.java)
            startActivity(intent)
        }

        if(intent.getStringExtra("filePath") != null)
        {
            tmpFile = intent.getStringExtra("filePath").toString()

            val file:File = File(tmpFile)

            val bitmap:Bitmap = BitmapFactory.decodeFile(file.absolutePath)

            imageView?.setImageBitmap(bitmap)

        }

    }
}